pkgs/build-support/compress-drv/default.nix

```diff
@@ -72,7 +68,11 @@ let
 in
 runCommand "${drv.name}-compressed" { } ''
   mkdir $out
-  (cd $out; ${xorg.lndir}/bin/lndir ${drv})
+
+  # cannot use lndir here, because it also symlinks directories,
```

Comment:
That comment is wrong. lndir never symlinks directories directly. Only if they are a symlink and then the input file is technically not a directory but a symlink to a directory.